### PR TITLE
To Do List Review

### PR DESCRIPTION
**Pull request by VanessaAoki**

### CODE REVIEW

**Comment by nelsinofrancisco** [comment]
# To do list review

Good job @VanessaAoki ! 🥳 🥳 you did a great job! I can't point out many errors in following best practices, but I can point out that your code is extremely DRY and good to read, so congratulations! Awesome Job.

Check out Inline comments I added some suggestions for you.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>To Do List</title>
+</head>
+<body>
+    <section class="to-do-container">
+        <h1>To Do</h1>
+        <form action="#" id="to-do-form">
+            <input type="text" name="text" id="to-do-input" placeholder="Add to your list...">
+            <button id="submit-button"><i class="fas fa-check"></i></button>
```
Comment (nelsinofrancisco) [inline index.html]: I suggest adding a type value to your button, as web hint flags me with this suggestion!

File: todo.js [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+/* eslint-disable func-names */
+/* eslint-disable no-plusplus */
+
+import './checkbox';
+
+let myToDo = [];
+
+if (localStorage.myToDo !== undefined) {
+  myToDo = JSON.parse(localStorage.myToDo);
+}
```
Comment (nelsinofrancisco) [inline todo.js]: Hello Vanessa! Your Project is dry, and I don't see problems with scalability. According to the KISS best practice principle, you could do a short circuit evaluation here, so I suggest using something like this: `let myToDo = JSON.parse(localStorage.myTodo) || [];`

File: todo.js [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+    buttonMenu.innerHTML = '<i class="fas fa-ellipsis-v"></i>';
+    buttonMenu.onclick = function () { showButtons(todo.index); };
+    buttonsDiv.appendChild(buttonMenu);
+  });
+}
+
+function updateLocalStorage() {
+  localStorage.myToDo = JSON.stringify(myToDo);
+  document.location.reload();
+  return false;
```
Comment (nelsinofrancisco) [inline todo.js]: In this function, you are always returning a boolean value, but I can't see where you are using this boolean value in your code, would it be better if you make this a void function?

**Comment by VanessaAoki** [comment]
Hi @nelsinofrancisco,

Thanks so much for the review.
I implemented all the requested changes, do you mind reviewing this pull request again?

thanks

**Comment by nelsinofrancisco** [comment]
# Project Approved!! 🥳 🥳 🥳 🥳 🥳 

Every suggestion was implemented you're good to go! It's time to merge it 🐸

**Pull request merged**